index.html

```diff
@@ -316,5 +328,127 @@ <h3 class="about-me__card__title">Skills</h3>
         </div>
       </div>
     </section>
+
+    <!-------CONTACT SECTION ------->
+    <section id="contact">
+      <div class="contact">
+        <div class="contact__container">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+
+          <form
+            class="contact__form"
+            action="https://formspree.io/f/xknaqawg"
+            method="POST"
+          >
+            <ul>
+              <li class="contact__form__element full-name">
+                <label class="label" for="full-name">Full name:</label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="full-name"
+                  placeholder="Full name"
+                  maxlength="50"
+                  required
+                />
+              </li>
```

Comment:
- [ ] As required for the project, kindly ensure that Name input has maxlength="30" and not maxlength="50"

Reply:
Ok, but notice this is for the “full name” feature. So, there exist full names over the 50 characters. So, I accept your change for the “name” and “last name” fields. But not for the full name because of my argument.

This template has two kind of forms, the first for the mobile version, that requires the “full name”, and the other for the desktop version that accepts both “name” and “last name”. Despite you're right, consider the logic of my comments.